pallets/liquidity-pools/src/message.rs

```diff
@@ -530,9 +565,17 @@ pub enum Message<BatchContent = BatchMessages> {
 		/// The amount of tranche tokens which should be redeemed.
 		amount: u128,
 	},
+	Forwarded {
```

Comment:
This will be a new message type on the solidity side as well right?

Reply:
Yes per https://kflabs.slack.com/archives/C06FRHCRQL9/p1723817828381219?thread_ts=1723816341.737369&cid=C06FRHCRQL9

Note: Not implemented yet in Solidity